vue/src/router/index.js

```diff
@@ -36,6 +36,7 @@ export default new Router({
       {
         path: '',
         name: 'Home',
+        redirect: '/boards/1',
```

Comment:
Router should not redirect to a specific board. boards/1 may not be the default board, or may not be present at all

Reply:
Solved